### DISCUSSION TITLE
Enhancement: reset button

### DISCUSSION THREAD

**Discussion by jayray123** [FunkinCrew/Funkin#1612] [Ideas]
a button that can reset the game if there is a bug or glitch

**Comment by GStudiosX2** [comment]
It wouldnt be possible to do that as u can't really detect bugs or Glitches but in the pause menu in the songs u can restart songs u click enter in the song btw

**Comment by Windows1Door2Daniel** [comment]
you can also press R to make a instant game over to start again
